tools/autograd/derivatives.yaml

```diff
@@ -421,8 +421,8 @@
   self: handle_r_to_c(self.scalar_type(), grad.conj() * other)
   other: handle_r_to_c(other.scalar_type(), grad * self)
 
-- name: _fused_dropout(Tensor self, float p, Generator? generator=None) -> (Tensor, Tensor)
-  self: _fused_dropout_backward(grad, result1, p)
+- name: native_dropout(Tensor input, float p, float scale, bool train) -> (Tensor, Tensor)
```

Comment:
note to myself: gradgrad in JIT is broken. We need to add an entry for native_dropout_backward here.